# Exception handling
'''Handling the errors without getting any errors in output'''
